### PR TITLE
Create MarkdownEditor and MarkdownView components

### PR DESCRIPTION
**Pull request by eric-zero-dev**
[Associated Notion Card](https://www.notion.so/zerotech/Markdown-components-a2a9a68ec6b5487eac1b4951742e8c6f)

-----

## 1. Pull request checklist

- [ ] Notion card has been moved to the Code Review column
- [ ] Notion card has a link to this PR
- [ ] A reviewer has been assigned to the Notion card

## 2. PR type
 
- [x] Feature

### CODE REVIEW

File: src/containers/DAO/DAOContainer.tsx [hunk 1]
```diff
@@ -52,7 +53,9 @@ const DAOContainer: React.FC<StakingContainerProps> = ({
 
 	return (
 		<main className={classNames(styles.Container, className)} style={style}>
-			{zna === '' ? <DAOList /> : <DAOPage />}
+			{zna === DAO_CREATE && <DAOCreate />}
+			{zna === '' && <DAOList />}
+			{zna !== DAO_CREATE && zna !== '' && <DAOPage />}
```
Comment (joonchoi0918) [inline src/containers/DAO/DAOContainer.tsx:58]: It's hard to read the conditional rendering. Can you update with more readable/understandable conditions? 💪

File: src/containers/DAO/pages/DAOCreate/DAOCreate.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+export const DAOCreate: React.FC = () => {
+	return (
+		<>
+			<h2>Markdown Editor Example</h2>
+			<MarkDownEditor text="" />
+
+			<hr />
+
+			<h2>Markdown Viewer Example</h2>
+			<MarkDownViewer text="Hello World" />
```
Comment (joonchoi0918) [inline src/containers/DAO/pages/DAOCreate/DAOCreate.tsx:13]: I know it is just example, but can we use longer and more complex markdown text as an example? 💪

**Review by joonchoi0918 — APPROVED** [review]
Looks good to me. I have a couple of comments and it will be easy.

File: src/containers/DAO/pages/DAOCreate/DAOCreate.tsx [hunk 3]
```diff
@@ -1,6 +1,20 @@
 import React from 'react';
```
Comment (colbr) [inline src/containers/DAO/pages/DAOCreate/DAOCreate.tsx:1]: Can we change this file to `CreateProposal.tsx`, as the page is for creating a proposal

**Review by colbr — CHANGES_REQUESTED** [review]
Looking nice, looks like a pretty handy library! One small filename change request

**Comment by colbr** [comment]
Closed as this was merged into another branch

**Pull request closed without merging**